language/libra-vm/src/data_cache.rs

```diff
@@ -74,7 +74,20 @@ impl<'block> StateView for StateViewCache<'block> {
                 Ok(remote_data) => Ok(remote_data),
                 // TODO: should we forward some error info?
                 Err(e) => {
-                    error!("[VM] Error getting data from storage for {:?}", access_path);
+                    // create a LibraLogger from the `data_view` in scope. This logger
```

Comment:
curious about opinions here, though it is a true pain to hook this up.
As the comment says though I wonder if it even matters to hook it up at all